From 454427d5ee983e26a907920874ca971584930eae Mon Sep 17 00:00:00 2001
From: "Enrico Weigelt, metux IT consult" <info@metux.net>
Date: Thu, 1 Feb 2024 16:25:11 +0100
Subject: [PATCH] glamor: glamor_debug.h: drop unused AbortServer() declaration

This really looks like a leftover from b861aad8e2fcf6fe1fae4f26abb650bb4eb499c6.

In any case, if that function shall become part of extension/driver API,
it should be declared with _X_EXPORT in some suitable header file - locally
declaring extern really isn't a good idea and just an invitation for subtle bugs.

Signed-off-by: Enrico Weigelt, metux IT consult <info@metux.net>
---
 glamor/glamor_debug.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/glamor/glamor_debug.h b/glamor/glamor_debug.h
index f64c448323..68189d484f 100644
--- a/glamor/glamor_debug.h
+++ b/glamor/glamor_debug.h
@@ -35,10 +35,6 @@
 #define GLAMOR_DEBUG_TEXTURE_DOWNLOAD         2
 #define GLAMOR_DEBUG_TEXTURE_DYNAMIC_UPLOAD   3
 
-extern void
-AbortServer(void)
-    _X_NORETURN;
-
 #define GLAMOR_PANIC(_format_, ...)			\
   do {							\
     LogMessageVerb(X_NONE, 0, "Glamor Fatal Error"	\
-- 
GitLab

